### PR TITLE
Augustus-3.0.2-goolf-1.4.10.eb (REVIEW)

### PR DESCRIPTION
**Pull request by pescobar**

### CODE REVIEW

**Comment by fgeorgatos** [comment]
Test report by @fgeorgatos
**FAILED**
Build succeeded for 0 out of 1
Linux debian 6.0.10, Intel(R) Xeon(R) CPU           L5640  @ 2.27GHz, Python 2.6.6
See https://gist.github.com/ed89ef42f1f194e307ca for a full test report.

File: easybuild/easyconfigs/a/Augustus/Augustus-3.0.2-goolf-1.4.10.eb [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+homepage = 'http://bioinf.uni-greifswald.de/augustus/'
+description = """ AUGUSTUS is a program that predicts genes in eukaryotic genomic sequences. """
+
+toolchain = {'name': 'goolf', 'version': '1.4.10'}
+
+source_urls = ['http://bioinf.uni-greifswald.de/augustus/binaries/']
+sources = [SOURCELOWER_TAR_GZ]
+
+dependencies = [('Boost', '1.53.0')]
```
Comment (fgeorgatos) [inline easybuild/easyconfigs/a/Augustus/Augustus-3.0.2-goolf-1.4.10.eb]: overall this looks good; just bump of possible the minor boost version

**Comment by fgeorgatos** [comment]
put this in REVIEW plz!

**Comment by migueldiascosta** [comment]
@pescobar, this PR is being closed for the following reason(s): no activity for > 6 months.
Please don't hesitate to reopen this PR or add a comment if you feel this contribution is still relevant.
For more information on our policy w.r.t. closing PRs, see https://easybuild.readthedocs.io/en/latest/Contributing.html#why-a-pull-request-may-be-closed-by-a-maintainer

**Pull request closed without merging**